recipes/argparse2tool/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "argparse2tool" %}
+{% set version = "0.4.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 749de9a1278e6c2795993a0d12e99492b7cf4e52d7862c9ef211f9ff17d8c52e
+
+build:
+  number: 0
+  entry_points:
+    - argparse2tool_check_path = argparse2tool.check_path:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - future
```

Comment:
Seeing https://github.com/hexylena/argparse2tool/commit/cdc93e5c1f164645045ecef40625e03b4806960d :
```suggestion
```
Plus, add `>=3.what-ever-is-appropriate` to the `python` dep in `run` as well as `host`.

Reply:
Good idea.